src/plugins/intel_gpu/tests/unit/test_cases/gather_gpu_test.cpp

```diff
@@ -2097,3 +2097,43 @@ TEST(gather_gpu_u8, 322_axisF) {
 TEST(gather_gpu_u8, export_import) {
     test_gather_gpu_u8_322_axisF<uint8_t>(true);
 }
+
+TEST(gather_opt_out, Baxis_gather_to_crop) {
+    auto& engine = get_test_engine();
+
+    auto input1 = engine.allocate_memory({ data_types::f32, format::bfyx, tensor{ 3, 2, 1, 2 } }); // Dictionary
+    auto input2 = engine.allocate_memory({ data_types::i32, format::bfyx, tensor{ 1, 1, 1, 1 } }); // Indexes
+    int64_t axis = 0;
+
+    set_values(input1, {
+        1.f, 2.f,  3.f,  4.f,
+        5.f, 6.f,  7.f,  8.f,
+        9.f, 10.f, 11.f, 12.f
+    });
+
+    set_values(input2, {
+        1
+    });
+
+    topology topology;
+    topology.add(input_layout("InputDictionary", input1->get_layout()));
+    topology.add(input_layout("InputText", input2->get_layout()));
+    topology.add(
+        gather("gather", input_info("InputDictionary"), input_info("InputText"), axis, ov::Shape{1, 2, 1, 2})
+    );
+    topology.add(reorder("reorder", input_info("gather"), format::bfyx, data_types::i8));
+
+    network network(engine, topology, get_test_default_config(engine));
+
+    network.set_input_data("InputDictionary", input1);
+    network.set_input_data("InputText", input2);
+
+    auto outputs = network.execute();
+
+    auto output = outputs.at("reorder").get_memory();
+    cldnn::mem_lock<int8_t> output_ptr(output, get_test_stream());
+
+    std::vector<int8_t> expected_results = {
+        5, 6, 7, 8
+    };
```

Comment:
what is the purpose of this test? It is just generated as gather, not crop..

Reply:
As I explained shortly, I wanted to check processing of B axis cropping by both gather and crop.
I also added similar test into crop_gpu_test.
Expected results of can_be_optimized are different.